migrations/sql-migrations/dynamoToSql/generateGem2sParamsHash.js

```diff
@@ -0,0 +1,44 @@
+const objectHash = require('object-hash');
+
+const DEFAULT_NA = 'N.A.';
+
+// Generates a params hash that is order agnostic
+const generateGem2sParamsHash = (project, samples, experiment) => {
```

Comment:
It this just a copy paste from the the UI?

Reply:
Yep